src/Workspaces/Remote/ServiceHub/Services/RemoteHostService.PerformanceReporter.cs

```diff
@@ -72,16 +72,16 @@ protected override void PauseOnGlobalOperation()
                                 // since it is telemetry, we hash analyzer name if it is not builtin analyzer
                                 m[nameof(badAnalyzerInfo.AnalyzerId)] = internalUser ? badAnalyzerInfo.AnalyzerId : badAnalyzerInfo.PIISafeAnalyzerId;
                                 m[nameof(badAnalyzerInfo.LOF)] = badAnalyzerInfo.LOF;
-                                m[nameof(badAnalyzerInfo.Mean)] = badAnalyzerInfo.Mean;
-                                m[nameof(badAnalyzerInfo.Stddev)] = badAnalyzerInfo.Stddev;
+                                m[nameof(badAnalyzerInfo.Average)] = badAnalyzerInfo.Average;
+                                m[nameof(badAnalyzerInfo.AdjustedStandardDeviation)] = badAnalyzerInfo.AdjustedStandardDeviation;
                             }));
                         }
 
                         // for logging, we only log once. we log here so that we can ask users to provide this log to us
                         // when we want to find out VS performance issue that could be caused by analyzer
                         if (newAnalyzer)
                         {
-                            _logger.TraceEvent(TraceEventType.Error, 0, $"[{badAnalyzerInfo.AnalyzerId} ({badAnalyzerInfo.Hash})] LOF: {badAnalyzerInfo.LOF}, Mean: {badAnalyzerInfo.Mean}, Stddev: {badAnalyzerInfo.Stddev}");
+                            _logger.TraceEvent(TraceEventType.Error, 0, $"[{badAnalyzerInfo.AnalyzerId} ({badAnalyzerInfo.AnalyzerIdHash})] LOF: {badAnalyzerInfo.LOF}, Mean: {badAnalyzerInfo.Average}, Stddev: {badAnalyzerInfo.AdjustedStandardDeviation}");
```

Comment:
:bulb: There are a bunch of inconsistent terms remaining at least in strings, parameters, and tuple names. These should be unified to the preferred .NET terminology.

Reply:
I either move back to original term or leave it as it is.